UncomplicatedCustomItems/Config.cs

```diff
@@ -12,117 +12,35 @@ public class Config : IConfig
-
-        [Description("List of custom armors")]
-        public Dictionary<int, SerializableCustomArmor> CustomArmors { get; set; } = new Dictionary<int, SerializableCustomArmor>()
-        {
-            {
-                0, 
-                new SerializableCustomArmor()
-                {
-                    Name = "Very heavy armor",
-                    Description = "Very heavy armor",
-                    Id = 0,
-                    Model = ItemType.ArmorHeavy,
-                    Info = new ArmorInfo()
-                    {
-                        BodyProtection = 99,
-                        HeadProtection = 99
-                    },
-                    SpawnPoint = new ItemSpawnPoint()
-                    {
-                        Location = Exiled.API.Enums.SpawnLocationType.Inside914,
-                        Chance = 100,
-                        Position = Vector3.one,
-                        Name = "Chipi chapa"
-                    }
-                }
-            }
-        };
-
-        [Description("List of custom weapons")]
-        public Dictionary<int, SerializableCustomWeapon> CustomWeapons { get; set; } = new Dictionary<int, SerializableCustomWeapon>()
-        {
-            {
-                0,
-                new SerializableCustomWeapon()
-                {
-                    Name = "Kalashnikov",
-                    Description = "Kalashnikov",
-                    Id = 0,
-                    Model = ItemType.GunAK,
-                    Scale = Vector3.one,
-                    Info = new WeaponInfo()
-                    {
-                        FireRate = 5,
-                        MaxAmmo = 35,
-                        Damage = 5
-                    }
-                }
-            }
-        };
-
-        [Description("List of custom items")]
-        public Dictionary<int, SerializableCustomItem> CustomItems { get; set; } = new Dictionary<int, SerializableCustomItem>()
-        {
-            {
-                0,
-                new SerializableCustomItem()
-                {
-                    Name = "Note",
-                    Description = "Just contains text from admin or other clown",
-                    Id = 0,
-                    Model = ItemType.KeycardGuard,
-                    Scale = new Vector3(1, 0.1f, 1),
-                    Info = new ItemInfo()
-                    {
-                        Command = string.Empty,
-                        Response = "Hello, welcome to UncomplicatedCustomItems by SpGerg!"
-                    }
-                }
-            },
-            {
-                1,
-                new SerializableCustomItem()
+                Id = 1,
+                Name = "MagicWeapon",
+                Description = "A magic weapon with an incredible firerate",
+                Item = ItemType.GunCOM18,
+                CustomItemType = CustomItemType.Weapon,
+                Scale = Vector3.one,
+                CustomData = new WeaponData()
```

Comment:
Im think it is must be work. Also yaml need change the IData values in runtime (how im know, yaml doesnt not support abstract classes, about interfaces im not sure)

Reply:
In fact I've used not abstract classes nor interfaces.
CustomItem and the other are fullyfied classes

Reply:
CustomData is interface

Reply:
You can be right, let me fix now